package.json

```diff
@@ -34,6 +35,7 @@
     "@typescript-eslint/eslint-plugin": "^2.20.0",
     "@typescript-eslint/parser": "^1.5.0",
     "@zerollup/ts-transform-paths": "^1.7.17",
+    "app-root-path": "^3.0.0",
```

Comment:
what's this for?

Reply:
To define path to the default place to save logs [here](https://github.com/jolocom/jolocom-sdk/pull/134/files#diff-cf09c68e036672514f763df6d940fd8635a7f66ef1fe5e29b6913f6773132a54R49)